gatsby-node.js

```diff
@@ -126,7 +128,7 @@ exports.sourceNodes = async (
   });
 };
 
-const processData = async (row, { createNodeId, createNode, store, cache }) => {
+const processData = async (row, { createNodeId, createNode, store, cache, defaultValues }) => {
   let data = row.fields;
```

Comment:
Would it be simpler to just merge via a spread operator up front here? Then it can travel through the process just like it had come from Airtable. Are there any downsides I am missing?

```suggestion
   let data = {...defaultValues, ...row.fields};
```

Reply:
Thinking about this further, my suggestion would run into the same issue in #32. I still think the spread operator may be more readable, but it would have to be moved to where you have your key iteration currently.

Reply:
I've got your point. In my opinion, the keys in the configuration in `gatsby-config.js` should be same with the column names in Airtable. User shouldn't know about key cleaning. It means `defaultValues` will be written with the same keys from Airtable, not cleaned ones. Does this make sense to you?
Not sure how to implement it, though.

Reply:
Unfortunately, the users will need to know about key cleaning as the cleaned key is the one that they will use to query with. Graphql has to use "cleaned" keys, so we don't have any choice. In which case, I think it makes sense that all things the user interacts with in "Gatsby-land" will be the cleaned key. We discussed warning which keys are cleaned in #32 as well, and directing them to documentation why this happens to help the user understand at least.

Reply:
Ah yeah, that makes sense. Understood.
Let me continue on this pr with your suggestion.